src/test/java/com/github/javafaker/PhotographyTest.java

```diff
@@ -0,0 +1,71 @@
+package com.github.javafaker;
+
+import com.github.javafaker.repeating.Repeat;
+import org.junit.Test;
+
+import static com.github.javafaker.matchers.MatchesRegularExpression.matchesRegularExpression;
+import static org.junit.Assert.*;
+
+public class PhotographyTest extends AbstractFakerTest{
+
+    @Test
+    public void testAperture() {
+        String value = faker.photography().aperture();
+        assertTrue(value.startsWith("f"));
+    }
+
+    @Test
+    public void testTerm() {
+        String value = faker.photography().term();
+        assertNonNullOrEmpty(value);
+    }
+
+    @Test
+    public void brand() {
+        String value = faker.photography().brand();
+        assertNonNullOrEmpty(value);
+    }
+
+    @Test
+    public void camera() {
+        String value = faker.photography().camera();
+        assertNonNullOrEmpty(value);
+    }
+
+    @Test
+    public void lens() {
+        String value = faker.photography().lens();
+        assertNonNullOrEmpty(value);
+    }
+
+    @Test
+    public void gengre() {
+        String value = faker.photography().gengre();
+        assertNonNullOrEmpty(value);
```

Comment:
```suggestion
    public void genre() {
        String value = faker.photography().genre();
        assertNonNullOrEmpty(value);
```

Reply:
@patou -  merci pour votre commentaire! And the suggestions.